api/client/client.go

```diff
@@ -341,9 +341,6 @@ func normalizeURL(addr string) (*url.URL, error) {
 	if u.Opaque != "" {
 		u.Host = net.JoinHostPort(u.Scheme, u.Opaque)
 		u.Opaque = ""
-	} else if u.Path != "" && !strings.Contains(u.Path, ":") {
-		u.Host = net.JoinHostPort(u.Path, "8888")
-		u.Path = ""
 	} else if u.Scheme == "" {
```

Comment:
While we're at it, should we fix the indentation of this if stanza and the ones below?